### PR TITLE
Voice Playgrounds brief mention

### PR DESCRIPTION
**Pull request by tommorris**
Adds a short section on Voice Playgrounds in the Voice Overview.

Also tidies up some sentences about the Nexmo CLI in same document.

### CODE REVIEW

File: _documentation/voice/voice-api/overview.md [hunk 1]
```diff
@@ -32,16 +32,25 @@ In this document you can learn about:
 
 ## Getting Started
 
+### Voice Playground
+
+In the [Nexmo Dashboard](https://dashboard.nexmo.com), you can try out the Voice API interactively in the Voice Playground. Once you are [signed up for a Nexmo account](https://dashboard.nexmo.com/signup), you can go to [Voice Playground](https://dashboard.nexmo.com/voice/playground) in the Dashboard (Voice ‣ Voice Playgrounds).
+
+When you use the Voice Playgrounds, you will be guided through the process of buying a phone number and assigning it to the Playground, then you can interactively test NCCOs in the browser and see the results. Voice Playgrounds also has a number of examples already implemented including IVR (with DTMF tones), streaming audio into a call, conference calling and connecting two users together.
```
Comment (leggetter) [inline _documentation/voice/voice-api/overview.md]: Would a bullet point list help highlight what it makes possible? But I agree we don't want to make this section too large.

Comment (tommorris) [reply _documentation/voice/voice-api/overview.md]: Can do.

**Comment by adambutler** [comment]
Looks like a good start.

I'd like to see us point to it alongside any code examples that overlap with the provided templates in the playground.

**Comment by tommorris** [comment]
@adambutler @leggetter now links to relevant tutorials and guides (and NCCO/API ref where a tutorial/guide isn't available).

**Comment by leggetter** [comment]
> I'd like to see us point to it alongside any code examples that overlap with the provided templates in the playground.

My concern with this is that it's another set of links that aren't to the Voice Playground.

I'm on the fence.

**Comment by leggetter** [comment]
@tommorris How about a brief mention on the NCCO page too?

**Comment by tommorris** [comment]
@leggetter Done, and rebased against `master`.

**Pull request merged**